src/OmniSharp.Http.Driver/app.config

```diff
@@ -62,6 +62,11 @@
                 <assemblyIdentity name="System.Composition.TypedParts" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
                 <bindingRedirect oldVersion="0.0.0.0-6.0.0.0" newVersion="6.0.0.0"/>
             </dependentAssembly>
+
+            <dependentAssembly>
+                <assemblyIdentity name="System.Memory" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
+                <bindingRedirect oldVersion="0.0.0.0-4.0.1.2" newVersion="4.0.1.2" />
```

Comment:
Why doesn't this version match up with the dependency version?

Reply:
Because the assembly version doesn't match the nuget package version.

Reply:
Might be worth a comment.